simpleflow/swf/process/worker/dispatch/dynamic_dispatcher.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+import importlib
+
+
+class Dispatcher(object):
+    def dispatch(self, name):
+        submodule_name, func_name = name.rsplit('.', 1)
+        submodule = importlib.import_module(submodule_name)
+        return getattr(submodule, func_name)
```

Comment:
👍  ; maybe we could make good use of it in `simpleflow.swf.helpers.find_activity()` later